providers/hns/hns_roce_u_hw_v2.c

```diff
@@ -916,9 +916,7 @@ int hns_roce_u_v2_post_send(struct ibv_qp *ibvqp, struct ibv_send_wr *wr,
 		attr_mask = IBV_QP_STATE;
 		attr.qp_state = IBV_QPS_ERR;
 
-		ret = hns_roce_u_v2_modify_qp(ibvqp, &attr, attr_mask);
-		if (ret)
-			*bad_wr = wr;
+		(void)hns_roce_u_v2_modify_qp(ibvqp, &attr, attr_mask);
```

Comment:
Why do you need casting to void?
Do you see any compilation warnings which require you to put it?

Reply:
Hi Leon, it is because of a warning reported by a inner static code analysis tool that all return values should be checked.
Is it ok to put the "(void)" here? If not, I will remove it. Thank you.

Reply:
Yes, please remove.

Reply:
OK, thank you.